src/libmongoc/tests/test-mongoc-loadbalanced.c

```diff
@@ -576,8 +576,10 @@ test_pre_handshake_error_does_not_clear_pool (void)
                                           NULL /* reply */,
                                           &error);
    /* A new connection is opened. */
-   request =
-      mock_server_receives_legacy_hello (server, "{'loadBalanced': true}");
+   request = mock_server_receives_hello_op_msg (server);
+   const bson_t *match_loadBalanced = tmp_bson ("{'loadBalanced': true}");
```

Comment:
Suggest moving reused variable up above outside the current "test block" to denote its value is not specific to this block.